script/retire-book.ts

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+import argv from 'yargs';
+import { makeUnifiedBookLoader } from '../src/app/content/utils';
+import { ARCHIVE_URL, REACT_APP_OS_WEB_API_URL } from '../src/config';
+import BOOKS from '../src/config.books';
+import createArchiveLoader from '../src/gateways/createArchiveLoader';
+import { getArchiveUrl } from '../src/gateways/createBookConfigLoader';
+import createOSWebLoader from '../src/gateways/createOSWebLoader';
+import updateRedirectsData from './utils/update-redirects-data';
+
+const booksPath = path.resolve(__dirname, '../src/config.books.json');
+
+const args = argv.string('bookId').argv as any as {
+  retiredBook: string,
+  redirectBook: string
+};
+
+const retireBook = async() => {
```

Comment:
@TomWoodward my thought here is that we will manually run this until we get to a place where we want to trigger it via ABL changes